steps/aws-device-farm-runner/0.0.5/step.yml

```diff
@@ -0,0 +1,157 @@
+title: Amazon Device Farm Runner
+summary: Amazon Device Farm Runner
+description: Deploys app to device farm and starts a test run. [Please see the wiki
+  for setup instructions](https://github.com/peartherapeutics/bitrise-aws-device-farm-runner/wiki).
+website: https://github.com/peartherapeutics/bitrise-aws-device-farm-runner
+source_code_url: https://github.com/peartherapeutics/bitrise-aws-device-farm-runner
+support_url: https://github.com/peartherapeutics/bitrise-aws-device-farm-runner/issues
+published_at: 2017-06-20T18:49:18.300482063-07:00
+source:
+  git: https://github.com/peartherapeutics/bitrise-aws-device-farm-runner.git
+  commit: 0d9e5ce0b7d7cceb6d04e2c57b0638f62d36dfe0
+host_os_tags:
+- linux
+- osx-10.9
+- osx-10.10
+type_tags:
```

Comment:
should be normalized, following this guide: https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md#step-grouping-convention

we did normalize your previous version, you can use that as an example: https://github.com/bitrise-io/bitrise-steplib/blob/master/steps/aws-device-farm-runner/0.0.4/step.yml#L20